### PR TITLE
Release notes for 24th April

### PR DESCRIPTION
**Pull request by shashankbarsin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+
+### Release notes
+
+* Bug Fixes
+  * Fixed CSI driver version display issue in Azure disk and file CSI Driver objects.
+  * Fixed bug where cloud-controller-manager was not deleting Node Object after deletion of VMSS instance.
+* Component Updates
+  * Azure Disk CSI driver has been updated to [1.16](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v1.16.0).
+  * Azure File CSI driver has been rolled back to [1.12](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.12.0) to avoid storage account creation every time when a new Azure file share volume is created.
+  * On AKS clusters of versions >= 1.22, nginx-ingress-controller images are updated from 1.0.5 to 1.2.0 to address [CVE-2021-25745](https://github.com/Azure/AKS/issues/2906) and [CVE-2021-25746](https://github.com/Azure/AKS/issues/2909).
```
Comment (PixelRobots) [inline CHANGELOG.md:21]: would be good to note that this affects anyone running the HttpApplicationRouting addon on their AKS cluster.

Comment (palma21) [reply CHANGELOG.md:21]: Not really, we manage app routing. Managed addons are patched automatically by us.

File: vhd-notes/AKSWindows/2019/17763.2803.220413.txt [hunk 2]
```diff
@@ -0,0 +1,177 @@
+Installed Features
```
Comment (Tatsinnit) [inline vhd-notes/AKSWindows/2019/17763.2803.220413.txt:1]: 💡 Please note: this file has missing this information which was consistent in previous release notes: https://github.com/Azure/AKS/blob/master/vhd-notes/AKSWindows/2019/17763.2686.220309.txt#L1-L15 

Not sure if that is a concious decision but just a little fyi. Thanks.

Comment (shashankbarsin) [reply vhd-notes/AKSWindows/2019/17763.2803.220413.txt:1]: That's build metadata related to the Azure DevOps pipeline and hosted agent used to build the image and is not related to the artifact produced, so we intentionally removed that metadata.

**Review by qpetraroia — APPROVED** [review]

**Review by palma21 — APPROVED** [review]

**Pull request merged**